data/persons.txt

```diff
@@ -1850,3 +1849,174 @@ ship "Modified Ladybug"
 	explode "huge explosion" 10
 	"final explode" "final explosion large"
 	description `This is an old war-torn Hai Ladybug with numerous modifications done on the hull. Apart from the additional turret mounts, the main adjustments on the vessel are three huge engine cells - two in the back replacing the old engine pods, and one in the front replacing the gun mounts. These push the Modified Ladybug to speeds only an insane captain would enjoy.`
+
+
+person "Patrol Team"
+	government "Author"
+	frequency 300
+	personality
+		mining harvests plunders
+	phrase
+		word
+#		Marksman
+			"Should you find another ship owned by Brick, tell them to work harder!"
+			"Should you find another ship owned by Zitchas, tell them they have a typo on line 9. They'll never see it coming!"
+			"If you don't know where to start in helping the game, take a nice look at the files. Sometimes, it helps!"
+			"If you're interested in helping out but don't have time, consider being a proofreader! You can take my job and I get to laze around doing nothing!"
+			"If you see something and it shoots at you, cheat your way to victory!"
+			"What did you just say about me you little-"
+			"The number of times someone breaks the fourth wall, it's mind-boggling!"
+			"Look, we aren't here for your money. We do all this for free, and look where that gets us."
+			"If you don't know where to start in helping the game, come and review some missions on GitHub. It helps!"
+			"If you're interested in helping out but don't have time, consider looking through the proposals on GitHub! Sometimes, a new perspective is needed!"
+			"What do you mean you don't have a save file for this Pull!?"
+			"Please, for the love of god, don't make an issue before checking if others have done the same."
+			"This statement is, intentionally wrote wrongly to make point."
+
+#		Roadrunner
+			"Typo detected. Initiating weapons engagement."
+			"Code non-functional. Destroy them."
+
+#		Saugia
+			"Yea... way too much detail. I can't see the cockpit anymore."
+			"Just don't go ham on details."
+			"These areas are looking much better, though. The contrast of the lesser details here and the more elsewhere is a lot nicer."
+			"I'm just trying to see this half that size and I'm losing a lot of detail."
+			"It's awesome, but far too detailed."
+			"Now there's a lot less detail conflicting with one another."
```

Comment:
I'd like to come back to these, I have a few ways of rewording them better. Instead of some specific lines, I'd reword them more as a generalized statement, ie.... "When modeling a ship, be careful on how much detail you give it!", etc.. (I'll come up with them.)